docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -277,6 +277,38 @@ The following are Azure CLI environment variables:
 | **AZURE_CONFIG_DIR**           | The global directory for configuration files, logs, and telemetry. If unspecified, this value defaults to `~/.azure.` |
 | **AZURE_EXTENSION_DIR**        | The directory containing extension installations. If unspecified, this value defaults to the `cliextensions` subdirectory within the global configuration directory. |
 
+## Error handling for Azure CLI in PowerShell
+
+You can run Azure CLI commands in PowerShell, as described in [Choose the right Azure command-line tool](choose-the-right-azure-command-line-tool.md). If you do, be sure you understand Azure CLI error handling in PowerShell. In particular, you can't use the `try` and `catch` keywords.
```

Comment:
There is nothing that prevents you from using `try` and `catch`. The only issue is that the Azure CLI does not create an exception that can be caught by PowerShell. You can only use the `$?` variable.

In a script you could use the `throw` keyword. So you can do something like this:

```powershell
try {
    az group create --name MyResourceGroup 
    if ($? -eq $False) {
        throw 'Group create failed.'
    }
}
catch {
    'Handle error here'
}
```